crux-core/src/crux/api/transaction/TransactionWrapper.java

```diff
@@ -0,0 +1,43 @@
+package crux.api.transaction;
+
+import clojure.lang.Keyword;
+import clojure.lang.LazySeq;
+
+import java.util.Date;
+import java.util.Map;
+
+public class TransactionWrapper {
```

Comment:
This might be better named `TxLogEntry`?

Given it's never submitted by a use,r I suspect this might also be better applied using the QueryState pattern - an interface to specify the data available, and a Clojure record to implement it - will save us the wrapping layer?

Reply:
Ah, ok, so the issue there will be with the tx-ops. :thinking:

Reply:
Atm, this is created from the path that required with-tx-ops to be true which is why the QueryState pattern might be a little fiddly.

I still think it will be doable, though. I'll have a try